avm/utilities/pipelines/staticValidation/psrule/.ps-rule/min-suppress.Rule.yaml

```
@@ -23,6 +23,8 @@ spec:
     - Azure.TrafficManager.Protocol
       # Azure Load Balancer specific
     - Azure.LB.Probe
+      # Azure Container App specific
```

```suggestion
      # Azure Managed-environment specific
```
While the test name doens't give it away, I think it makes sense to connect this to the module in which it is relevant, that is, managed-environment.